### PR TITLE
font size is 12px

### PR DESCRIPTION
**Pull request by crmorford**
:|

### CODE REVIEW

File: lib/widgets/tutorial-tooltip/index.styl [hunk 1]
```diff
@@ -229,7 +229,7 @@ $tooltip-distance = 12px
     .next, .done
       color blue-100
       cursor pointer
-      font-size 14px
+      font-size 12px
```
Comment (zakj) [inline lib/widgets/tutorial-tooltip/index.styl]: intentional for that to apply to all our existing onboarding tooltips too?

Comment (crmorford) [inline lib/widgets/tutorial-tooltip/index.styl]: hm i added this for notif 3.0 so i assumed we were the only ones using this new feature of it. maybe that was incorrect.

Comment (zakj) [inline lib/widgets/tutorial-tooltip/index.styl]: oh wow I think you're right! how did we not have a "next" before?!

**Comment by zakj** [comment]
0.75.5

**Pull request closed without merging**